src/components/card/card-components.tsx

```diff
@@ -70,50 +70,44 @@ export interface CardProps extends HTMLAttributes<HTMLDivElement> {
   isSelected?: boolean // Does the card have the blue selected border
 }
 
-export const handleToggleContextMenu = (
```

Comment:
Weird formatting - can you run `yarn lint --fix` just to check please

Reply:
Have done. No change 👍🏼

Reply:
If you look in your node_modules, which version of prettier do you have? I get 3.2.5 - maybe you need to do a yarn install?